bin/common.sh

```diff
@@ -73,8 +73,18 @@ function download_mysql_jdbc_jar(){
     fi
     MYSQL_JAR_URL="https://repo1.maven.org/maven2/mysql/mysql-connector-java/${MYSQL_VERSION}/mysql-connector-java-${MYSQL_VERSION}.jar"
   fi
+
   echo "Downloading mysql jdbc jar from ${MYSQL_JAR_URL}."
-  wget ${MYSQL_JAR_URL} -P "${BIN}/../lib"
+  if type wget >/dev/null 2>&1; then
+    wget ${MYSQL_JAR_URL} -P "${BIN}/../lib" --no-check-certificate
+  elif type curl >/dev/null 2>&1; then
+    curl -o "${BIN}/../lib/mysql-connector-java-${MYSQL_VERSION}.jar" ${MYSQL_JAR_URL}
```

Comment:
Could we use "${SUBMARINE_HOME}/lib/mysql-connector-java-${MYSQL_VERSION}.jar"

Reply:
This is a good question.
But using the `SUBMARINE_HOME` variable, you must define the path in `submarine-env.sh`.
I think it may be easier to use `SUBMARINE_HOME` in the future.

Reply:
`SUBMARINE_HOME ` already define in common.sh
https://github.com/apache/submarine/blob/7576a5e300b4659c3a8a36c035e125d7a5c7f44e/bin/common.sh#L30

Reply:
Okay, thanks for your reminder.